src/skin/js/firstRun.js

```diff
@@ -15,4 +15,45 @@ $(function () {
       });
     }
   });
+
+  function hideNudgeOverlay() {
+    $("#pin-nudge").fadeOut();
+    $("#overlay").fadeOut();
+  }
+
+  // Don't show the pin nudge in Firefox because extensions are pinned automatically
+  // chrome.action is only available in MV3 and chrome.browserAction is only available in <= MV2
+  // chrome.browserAction.getUserSettings doesn't exist in MV2 but does in Firefox
+  let isChromiumMv3 = chrome.action;
```

Comment:
`!!chrome.action` to make a boolean

Reply:
[Booleans should get snake case names](https://github.com/EFForg/privacybadger/blob/master/doc/develop.md#style-guide), same as integers.